CHANGELOG.md

```diff
@@ -61,6 +61,7 @@ All notable changes to this project will be documented in this file.
     * #### Added
         * `get_channel_names()`
     * #### Changed
+        * Fix [#1770](https://github.com/ni/nimi-python/issues/1770): fetch(), read(), and friends return wrong data when called with channel ranges on multi-instrument niscope session.
```

Comment:
remove `niscope` since it is under the `niscope` section, also because it isn't clear if it should be "NI-SCOPE session" or `niscope.Session`.

Reply:
Removed. I left the title of the actual issue unchanged, though.